psatoken_test.go

```diff
@@ -426,13 +393,21 @@ func TestClaims_sign_and_verify(t *testing.T) {
 }
 
 func makeClaims() Claims {
-	profile := PSA_PROFILE_1
+	profile, _ := eat.NewProfile(PSA_PROFILE_2)
```

Comment:
Shall we not test for error returned from eat.NewProfile()?

this way any incompatibility introduced can be caught...

Reply:
sgtm, will do!

Reply:
Done in b4b3f31